core/modules/replica/tools/qserv-replica-job-verify.cc

```diff
@@ -31,14 +30,15 @@
 // Qserv headers
 #include "replica/VerifyApp.h"
 
+using namespace std;
```

Comment:
I wonder if this should be alphabetized, and go below `using namespace lsst...`?

Reply:
Good question. Though, I've found nothing on this subject in the Style Guide:
https://developer.lsst.io/cpp/style.html#appendix-on-using-using